src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -181,8 +182,12 @@ private MessageConsumer newAcceptStream(
         final long applicationRef = begin.sourceRef();
         final String applicationName = begin.source().asString();
         final long authorization = begin.authorization();
-        final OctetsFW extension = begin.extension();
-        final TlsBeginExFW tlsBeginEx = extension.get(tlsBeginExRO::wrap);
+        // Ignoring extension data until there is a way to find a specific extension in the whole
+        // extension data
+        // final OctetsFW extension = begin.extension();
+        // final TlsBeginExFW tlsBeginEx = extension.extension.get(tlsBeginExRO::wrap);
```

Comment:
Please remove these comments and refer to issue number reaktivity/nukleus-tls.java#47 instead.